### DISCUSSION TITLE
Welcome to Community-Governance Discussions!

### DISCUSSION THREAD

**Discussion by Phyloiota** [iota-community/Community-Governance#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Phyloiota** [comment]
Hey - my Name is Holger. I am a German, 42 years old and live in Bali / Indonesia. I worked in the chemical Industry in Germany for many many years in different functions and projects. I left Germany together with my wife in 2012 to work as a dive Instructor in Asia. Since 7 years i am now based in Bali.
I know IOTA since beginning of 2018 and joined the IOTA Discord in May 2018. The first years i was just a fascinated observer, but since the implementation of the X-Teams i became more involved and found fun in actively participating in the project.

I really hope we can start a working and productive group here and take this great chance to empower the IOTA community
My Discord is Phylo#2233 so feel free to contact me there.

**Reply by ricardroyo** [reply]
Man this like a dream come true! I went to Bali for my honeymoon and I got to do some diving in the Pemuteran area. What a great place!

**Reply by Phyloiota** [reply]
Ha really? - Pemuteran is where it all started for me - i learned diving there 13 years ago and fell in love with Bali. Now i live since 7 years in Amed - east coast - even more beautiful.

**Reply by ricardroyo** [reply]
Yes Pemuteran was actually my first dive too. I did not have the PADI back then so it was just a "test dive" but I loved the experience and when I came back I got the PADI in Islas Medas (north east of Spain). I did some diving for a while before children, but now I haven't done it in a long time and I don't even know if I would remember how to do anything TBH. 
I'm looking forward to going back to Bali one day with my children. Maybe when they are older though.

**Reply by Phyloiota** [reply]
> Yes Pemuteran was actually my first dive too. I did not have the PADI back then so it was just a "test dive" but I loved the experience and when I came back I got the PADI in Islas Medas (north east of Spain). I did some diving for a while before children, but now I haven't done it in a long time and I don't even know if I would remember how to do anything TBH.
> I'm looking forward to going back to Bali one day with my children. Maybe when they are older though.

Haha so cool. 
So hereby offered a personal refresh and come back into diving by me if you ever will be able to make it back to Bali!

**Reply by ricardroyo** [reply]
Thanks for the offer, I will certainly let you know! I can already see my wife rolling her eyes when I tell her that I want to meet a IOTA friend during our future Bali holidays XD

**Comment by datahomelab** [comment]
Hello Everyone. My name is Chris, and I'm 53 years old (Aug 23, 1967). I am **Canadian/French/Argentinean** citizen, and I live in Seremban, Malaysia. I have been working in the Aviation Industry as an Airline Captain for the last 25 years. Before doing this, I was a Software Developer in Canada. I worked as a SYSAdmin and Programmer for the Groupe Option Retraite, a financial firm based in Montreal, Quebec. This was my first job out of Concordia University, where I got my **BSc in Applied Math and Computer Science**. However, flying was my childhood dream, so I left the programming world behind to become an Airline Pilot at the age of 27. I can communicate fluently in English, Spanish and French, and understand Italian and Portuguese. I have basic understanding of Chinese, Korean and Bahasa Malaysia. I have lived in Asia for the last 17 years.

I became interested in IOTA back in 2018, and I joined Discord in July that year. As Holger, I was a very interested observer. I have been also on the Telegram Groups since then too. Because aviation was taking much of my time and energy, and because my career had me moving from Abu Dhabi to Malaysia in 2018, I barely had time to participate. However, since COVID, I have been grounded, and maybe permanently. Therefore, I have found new energy to go back to my old profession. Since last year I have been learning DEVOPS: Docker, Kubernetes, Ansible, and how to develop software in this new and more complex world. I have taken courses on Swift, HTML and JS. Although my brain is not as fast as it used to be, I can still understand the new paradigms... My next task: **Learning Rust**.

I think myself as a bridge across cultures... I was born in South America, I became an adult in North America, and because of my work as a pilot, I have lived in every continent (apart from Antartica...). As an Airline Captain, I have managed teams at the leading edge of my profession on a daily basis, and these teams have become more and more multi-cultural as my experience grew. Along the way, I became a Human Factors in Aviation specialist, essentially, trying to figure out why accidents and incidents happen in Aviation, and finding ways to mitigate these risks in the future.

Well, that's it for me... I hope to meet interesting people, learn new things, and essentially be part of a productive discussion. Sorry for the long intro, but it's tough to condense 30 years in one paragraph. I am chrispilot#7770 in Discord.

**Comment by ricardroyo** [comment]
Hi there! My name is Ricard, I live in Barcelona, Spain with my wife and 2 kids. I'm a computer engineer, although life has brought me into other directions and even though I'm still an engineer at hard, it is not my day to day any more.

I studied my engineering degree at the Autonomous University of Barcelona and met my wife while being an exchange student at University of California in Davis. After graduation, I worked at a computer vision software company and became the CTO of an Internet startup in the automotive industry that ended up being acquired by a larger competitor. Since then, I've been involved in the local LegalTech industry and cofounded a small company to offer online low-cost legal services with a friend I made while doing mountain bike, my other passion.

I became aware of IOTA in 2017 and even though I missed the crowdsale I felt in love with the vision and the community. In 2018 I decided to get deeper into the rabbit hole, joined IEN and without previous experience, I organized a couple of Meetups taking advantage of the local startup ecosystem in Barcelona (https://www.youtube.com/channel/UCaX9lGm1cTH2ibxsB_Cba1w). IOTA to me has always been a great source of brain food and intelligent people. Being part of the community has allowed me to broaden my understanding of the whole crypto industry, make friends and enjoy great conversations. Due to COVID and my daughter (child #2) being born, I don't have as much time now as I used to have when I organized those Meetups, but I've tried to keep up with the updates and play with the node software (my node is https://tangle.monster). I'm planning to still be involved in the community as much as life allows me. 
 If you want to reach me you can find me under ricardroyo#1508 in the IOTA Discord

**Reply by datahomelab** [reply]
I remember talking to you back in 2018... Buenas y encantado de conocerte!

**Reply by ricardroyo** [reply]
Really? Sorry I don't remember you at all. Nice to meet you again!

**Reply by ricardroyo** [reply]
Gracias :)

**Comment by sascha1337** [comment]
Amazing place to be - @Phyloiota is my 'neighbour', 45 min with the speedboat to the next island and we could meet - both permanent here - we met on twitter because we found out we both speak indonesian :D But im native and born in region cologne, Germany.

After working with Fujitsu as external IOTA expert on that 2018 Hannover Trade Fair IOTA Audit Trail #HMI18 showcase, i did want to have a break from hard work and visited some cryptofriends in Bali, out of convenience i"ve chosen to go there on-way-ticket with 24 years, so i can book the ticket back, when i feel like it, or the prices are great. Never happened. 

3 Years later still here, married, and got father. I did cover cost with my automated affiliate commission <-> selfwritten global load balanced CDN style analytics and traffic source management system since 2013.

But with corona and unexpected cost wiping my finances and emptying my crypto wallets to 'survive',  i had to deal with 90% breakdown of incomestream, which was not easy at all. Worked in generic jobs as fullstack dev, but never really been happy with those trivial tasks, i had IoT, IOTA, Smart Contracts, Hardware, Code in my head, not writing yet-another-crm for managing donations or setting up WordPress websites. 

Now full-time on IOTA and community work / working with Zignar Technologies on embedded microprocessors, Agricultural IoT solutions that use the full spectrum of IOTA to optimize production processes with sensordata,  making the data more trustworthy and pushing Interoperability engineering forward, as we are going to need it for Industry 4.0 and smart cities / sharing economy - M2M.   

But reality draws me back every X days or weeks, forcing me away from working on IOTA -  simply to just cover the basic needs - trivial tasks or freelance jobs, that reaaaally are not fun - due to old tech and repetitive. - overtime hours, because i also want to keep progress flowing on all fronts. Trapped in 'paradise'. 

2018-2019 drafted some really advanced DAO governance system on IOTA with some other smart people i met here, from whitepaper until PoC demo UI's that used MAM and IOTA payloads as backend, even got support by number 1 best, well known, german speaking architect of Bali - he did draw and render drafts of an IOTA IoT-Lab Community DAO Campus in Bali, with papers of some non-investor-type profit oriented business model - DAO & Community Funds on Smart Contracts - but we were far ahead of the time. We thought ISCP will drop earlier.

So much more to talk about, or meantion, but thats some small summary of my last years on the #rollercoaster IOTA since 2017.

**Comment by basvansambeek** [comment]
Hi, I'm Bas. I live in the Netherlands. I've just moved, so a lot of my time right now is spent on making our house a home. I've been in the IOTA community since 2017, visiting meetups across Europe and helping answer questions on various platforms.

I've been self-employed since 2005, and my specialism is positioning and communication strategy. I have a technical/engineering background, and in my work I help tech companies find a way to make their technology fit the users' demands and expectations.

My interest within the IOTA ecosystem lies on the human side of the machine economy. Right now there's a lot of work being done on making the tech work, but where I hope to contribute is to make the tech work for humans. On the blog IOTA-untangled, which I maintain with my wife, we explore the implications of a machine economy beyond the technology itself. (It's been a bit quiet in corona times. We hope to pick up the pace again soon™.)

To me engineering and psychology are two sided of the same coin in marketing. Engineers (understandably) focus on making the tech work, but because that is such a challenging task, the human side might not get the attention it requires. Once a product reaches the proof of concept stage, I can start adding value by tweaking the product into being more user friendly. Here I tend to focus on the irrational side of humans, who tend to just not make the logical decisions you could reasonably assume while designing a product.

With Chrysalis being delivered and Nectar on the horizon, for me the fun is finally starting. I can't wait to see where the development will lead us.

**Comment by ACC30** [comment]
my name is Alexander, I'm 36. Russia.  Astrakhan.  I have been familiar with the iota since 2017. Even then I was attracted by the idea of ​​a distributed ledger that was low-cost in terms of power consumption.  one day the world will realize that wasting resources on maintaining expensive books will become obsolete).  I got distracted.  I am a geologist by education, but I have always dreamed of connecting my life with IT.  The work is related to geology.  I started my career with geological and hydrodynamic modeling.  I am now a middle manager.  to my great regret, I have a rather superficial knowledge of the English language, which prevents me from developing in IOTA.  Studying materials and discussions take me a lot of time, but I try to keep abreast of all events.  the chrysalis catalyst stimulates me.  I want to be useful to the community, but so far I have not found a use for myself.  ACC # 7790 discord.

**Comment by dudeamir** [comment]
Hi, my name is Amir from Iran. I moved in to Germany in 2011, doing my masters in Freiburg, then my PhD (to be defended in 2021) in TU-Berlin in microelectronics. 
I got to crypto around 2016, mostly speculating. 2020, decided to leave my apartment and moved in a Van and traveled Europe during pandemic, wrote down my doctor thesis and as I was sure, I dont want to go work in microelectronics, was searching to see where can i find my passion. At beginning, I was thinking to open a Sustainability Cafe in Berlin to promote the technologies available (water harvesting, vertical farming, energy harvesting,...) in small scale and implement it in the Cafe, and on my research I came to think of installing sensors in wearables to measure any data can be harvested from costumers body and giving them a discount on their cafe. Then started to search with technology can enable it and IOTA was the obvious answer. 
After seeing IEEE smart city society also introduce IOTA as the most optimum solution from perspective of engineers, it made up my mind IOTA is the tech which has high potential. Ever since, I started working toward a startup to utilise IOTA tech in a health solution bridging patients and data scientists. Slight twist from opening a cafe in Berlin, but definitely more exciting to build sth on the edge of what technology can enable.
I find this community, highly civilised and educated, therefore wish to contribute more to see IOTA prosper. Prosperous IOTA enables other applications such as what I pursue, therefore my incentives are aligned with success of IOTA.
